### PR TITLE
Update 15 Changes

### PR DESCRIPTION
**Pull request by dfeinzimer**
See #177

### CODE REVIEW

File: Documentation/TimeSlider/README.md [hunk 1]
```diff
@@ -56,33 +56,33 @@ User can control whether to show `Slider` and/or `Playback Buttons` using `isSli
...
+timeSlider.labelMode = .ticks
+timeSlider.addTarget(self, action: #selector(TimeSliderExample.timeSliderValueChanged(timeSlider:)), for: .valueChanged)
+view.addSubview(timeSlider)
+
+// Add constraints to position the slider
+let margin: CGFloat = 10.0
+timeSlider.translatesAutoresizingMaskIntoConstraints = false
+timeSlider.bottomAnchor.constraint(equalTo: mapView.attributionTopAnchor, constant: -margin).isActive = true
+
+if #available(iOS 11.0, *) {
```
Comment (yo1995) [inline Documentation/TimeSlider/README.md:70]: IMO with the minimum required Runtime SDK version as 100.15.0, the min iOS version should be iOS 13, so this API check can be removed?

Comment (mhdostal) [reply Documentation/TimeSlider/README.md:70]: Nice catch!

File: Documentation/JobManager/README.md [hunk 2]
```diff
@@ -7,35 +7,34 @@ The Job Manager is a toolkit component that you can just plug into the applicati
...
+	return
+    }
 
-            // make sure we are still around...
-            guard let strongSelf = self else {
-                return
-            }
+    // make sure task does not get released
+    guard let strongTask = task else{
+	return
```
Comment (zkline101) [inline Documentation/JobManager/README.md:21]: The indentation for this return statement looks off. Same with the guard statement above.

**Review by zkline101 — COMMENTED** [review]
Thanks for cleaning up some of that doc! I have one small comment.

File: Documentation/JobManager/README.md (hunk 2 above)
Comment (dfeinzimer) [reply Documentation/JobManager/README.md]: Good catch. Looks like there were tabs instead of spaces, and the wrong amount

**Review by zkline101 — APPROVED** [review]

**Pull request merged**